pages/src/docs/src/Defs.js

```diff
@@ -200,54 +238,56 @@ var MemberDef = React.createClass({
 
 exports.MemberDef = MemberDef;
 
-
 function functionParams(info, params, shouldWrap) {
-  var elements = Seq(params).map(t => [
-    t.varArgs ? '...' : null,
-    <span className="t param">{t.name}</span>,
-    t.optional ? '?: ' : ': ',
-    <TypeDef
-      prefix={t.name.length + (t.varArgs ? 3 : 0) + (t.optional ? 3 : 2)}
-      info={info}
-      type={t.type}
-    />
-  ]).interpose(shouldWrap ? [',', <br />] : ', ').toArray();
-  return shouldWrap ?
-    <div className="t blockParams">{elements}</div> :
-    elements;
+  var elements = Seq(params)
+    .map(t => [
+      t.varArgs ? '...' : null,
+      <span className="t param">{t.name}</span>,
+      t.optional ? '?: ' : ': ',
+      (
+        <TypeDef
+          prefix={t.name.length + (t.varArgs ? 3 : 0) + (t.optional ? 3 : 2)}
+          info={info}
+          type={t.type}
+        />
+      )
+    ])
+    .interpose(shouldWrap ? [',', <br />] : ', ')
+    .toArray();
+  return shouldWrap
+    ? <div className="t blockParams">{elements}</div>
+    : elements;
 }
 
 function callSigLength(info, module, name, sig) {
-  return (
-    (module ? module.length + 1 : 0) +
-    name.length +
-    funcLength(info, sig)
-  );
+  return (module ? module.length + 1 : 0) + name.length + funcLength(info, sig);
 }
 
 function funcLength(info, sig) {
-  return (
-    (sig.typeParams ? 2 + sig.typeParams.join(', ').length : 0) +
-    2 + (sig.params ? paramLength(info, sig.params) : 0) +
-    (sig.type ? 2 + typeLength(info, sig.type) : 0)
-  );
+  return (sig.typeParams ? 2 + sig.typeParams.join(', ').length : 0) +
+    2 +
+    (sig.params ? paramLength(info, sig.params) : 0) +
+    (sig.type ? 2 + typeLength(info, sig.type) : 0);
 }
 
 function paramLength(info, params) {
-  return params.reduce((s, p) =>
-    s +
-    (p.varArgs ? 3 : 0) +
-    p.name.length +
-    (p.optional ? 3 : 2) +
-    typeLength(info, p.type),
+  return params.reduce(
+    (s, p) =>
+      s +
+      (p.varArgs ? 3 : 0) +
+      p.name.length +
+      (p.optional ? 3 : 2) +
+      typeLength(info, p.type),
```

Comment:
the indentation is nice, before if was unclear at a glance that there was a second argument :)